homeassistant/components/rejseplanen/sensor.py

```diff
@@ -115,18 +119,17 @@ def device_state_attributes(self):
         if len(self._times) > 1:
             next_up = self._times[1:]
 
-        return {
-            ATTR_DUE_IN: self._times[0][ATTR_DUE_IN],
-            ATTR_DUE_AT: self._times[0][ATTR_DUE_AT],
-            ATTR_TYPE: self._times[0][ATTR_TYPE],
-            ATTR_ROUTE: self._times[0][ATTR_ROUTE],
-            ATTR_DIRECTION: self._times[0][ATTR_DIRECTION],
-            ATTR_STOP_NAME: self._times[0][ATTR_STOP_NAME],
-            ATTR_STOP_ID: self._stop_id,
+        attributes = {
             ATTR_ATTRIBUTION: ATTRIBUTION,
             ATTR_NEXT_UP: next_up,
+            ATTR_STOP_ID: self._stop_id,
         }
 
+        if self._times[0] is not None:
```

Comment:
This little change causes device attributes not to be consistent.
If the time is unknown, it will result in those additional attributes not being set.

We prefer to have a static/consistent set of attributes. If the attribute value is unknown, `None` can be returned as the attribute value in those cases.

Reply:
Oh, I've been told the opposite before; that null attributes should not be returned, which is why I made it like that.

I follow the same thought further down with track and real time departure, can I just return `None` there as well? That would simplify the code a bit. :)

Reply:
Our current standard is to not set the attribute item at all if the value is `None`.

Reply:
I'm sorry, I missed that / mixed that up, which is totally my bad @jnewland

Reply:
Okay, I'll revert my change then.

Reply:
@frenck I'm not sure, but I think you need to approve this before it can be merged?